src/firefly/js/templates/lightcurve/LcViewer.jsx

```diff
@@ -241,10 +241,19 @@ export const UploadPanel = ({initArgs, name= 'LCUpload'}) =>{
 
 
 UploadPanel.propTypes = {
-    name: PropTypes.oneOf(['LCUpload']),
+    name: PropTypes.string,
     initArgs: PropTypes.object
 };
 
+UploadPanel.defaultProps = {
+    name: 'LCUpload'
+};
+
+// UploadPanel.propTypes = {
+//     name: PropTypes.oneOf(['LCUpload']),
+//     initArgs: PropTypes.object
+// };
```

Comment:
`defaultProps` is deprecated.  Use default parameters instead.
Also, no need to add commented out code.  It's already in history.